### ISSUE TITLE
WCAG 3.1.1: <html> element must have a lang attribute (html) (docs pages)

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2286]
#### Issue
`<html> element must have a lang attribute` ([html-has-lang](https://dequeuniversity.com/rules/axe/3.2/html-has-lang?application=msftAI))
#### Target application
[Documentation - Microsoft MakeCode](https://makecode.microbit.org/docs)
#### Element path
html
#### Snippet
`<html xmlns="http://www.w3.org/1999/xhtml" xmlns:og="http://ogp.me/ns#" xmlns:fb="http://www.facebook.com/2008/fbml">`
#### How to fix
    Fix any of the following:
      The <html> element does not have a lang attribute
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

**Comment by jwunderl** [comment]
@pelikhan I just need to add ``lang="@locale@"`` [here in docs.html](https://github.com/microsoft/pxt/blob/edc3ddb079ec6fd5b00de6fdd1b4b02c878f7091/docfiles/docs.html#L2) to make it patch in the current locale to fix this, right? I don't know too much about how that patching works because I don't have access to the backend

**Comment by abchatra** [comment]
Closing 2 year old bugs.
